src/test/java/io/github/christophermanahan/carnitas/HeadersTest.java

```diff
@@ -13,4 +16,14 @@ void itCanConstructAContentLengthHeader() {
 
         assertEquals(Headers.CONTENT_LENGTH + length, header);
     }
+
+    @Test
+    void itCanConstructAnAllowHeader() {
+        List<String> methods = List.of(Router.GET, Router.HEAD);
+
+        String header = Headers.allow(methods);
```

Comment:
How many more methods will be added to `Headers`?

Reply:
@damonkelley now using a Request.Method enum!